accounts/templates/account/login_errored_cancelled.html

```diff
@@ -10,13 +10,13 @@
 <div class="container">
   <div class="jumbo" style="padding-top: 25px">
       <h1 style="font-size: 48px; text-align: center; font-weight: 300;">
-          Login Unsuccessful
+          {% trans "Login Unsuccessful" %}
       </h1>
       <h4 style="text-align: center; font-weight: 300;">
-          If this should not have occurred and the issue persists, please report it to <a href="mailto:support@osf.io">support@osf.io</a>.
+          {% trans 'If this should not have occurred and the issue persists, please report it to <a href="mailto:support@osf.io">support@osf.io' %}</a>.
```

Comment:
same comment here - I don't know if this causes any issues but should probably either include the closing tag or skip the link entirely